apps/frontend/core-dapp/src/components/PpoRewardSummaryRecord.tsx

```diff
@@ -17,46 +18,43 @@ const Wrapper = styled.div`
   justify-content: flex-end;
 `
 
-/**
- * We are moving from standalone `ppoReward` and `ppoValue` to {@link PpoReward}
- * @deprecated use {@link PPORewardSummaryRecord} instead.
- */
-export const PPORewardSummaryRecordOld: FC<{
-  label?: string
-  ppoReward: string | undefined
-  ppoValue?: number
-}> = observer(({ label = 'PPO Reward', ppoReward, ppoValue }) => {
-  // if reward is undefined, show loading skeleton.
-  // if has reward, show reward UI.
-  // If no reward, show nothing
-  if (ppoReward !== undefined && +ppoReward <= 0) {
+export const PPOReward: FC<{
+  displayValueInEth?: boolean
+  reward: PpoReward | undefined
+}> = observer(({ displayValueInEth = true, reward }) => {
+  const { ppoTokenStore } = useRootStore()
+
+  const ppoReward = reward === undefined ? undefined : ppoTokenStore.formatUnits(reward.inPpo)
+  const ppoValue = reward === undefined ? undefined : +formatEther(reward.inEth)
+
+  if (ppoReward === undefined) {
+    return <Skeleton height="22px" width="64px" />
+  }
+
```

Comment:
Not a blocker but I think if you rearrange these, ppoValue won't need the undefined check and in the `displayEth` code below you'll have something cleaner: `displayEth(ppoValue)`

```suggestion
export const PPOReward: FC<{
  displayValueInEth?: boolean
  reward: PpoReward | undefined
}> = observer(({ displayValueInEth = true, reward }) => {
  const { ppoTokenStore } = useRootStore()

  const ppoReward = reward === undefined ? undefined : ppoTokenStore.formatUnits(reward.inPpo)
    if (ppoReward === undefined || reward === undefined) {
    return <Skeleton height="22px" width="64px" />
  }
  
  const ppoValue = +formatEther(reward.inEth)

```

Reply:
Good catch! Definitely cleaner